### PR TITLE
feat: allow to define route log level

### PR DESCRIPTION
**Pull request by leandroandrade**
The purpose of this PR is to allow to define a specific log level in scenarios when we want to `silent` all logs of the swagger route.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/mode/dynamic.js [hunk 1]
```diff
@@ -29,13 +29,15 @@ module.exports = function (fastify, opts, done) {
     const initOAuth = opts.initOAuth || {}
     const staticCSP = opts.staticCSP
     const transformStaticCSP = opts.transformStaticCSP
+    const logLevel = opts.logLevel || 'info'
```
Comment (mcollina) [inline lib/mode/dynamic.js:32]: I would not set a default value for this, let it be managed by fastify itself

**Review by mcollina — CHANGES_REQUESTED** [review]
Thanks for opening a PR! Can you please add a unit test?

**Review by climba03003 — APPROVED** [review]
LGTM.

I believe the next step after this PR should be group all those parameter into a single object instead of mapping it one by one.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**